front/src/modules/users/components/Avatar.tsx

```diff
@@ -13,6 +14,7 @@ type OwnProps = {
   placeholder: string;
```

Comment:
let's introduce a size enum (see IconButton for example)
Small | Medium | big

And let's compute height, width and font-size properties from this (see IconButton for example)

Reply:
what should be the mapping? for iconbutton its

`      case 'large':
        return '32px';
      case 'medium':
        return '24px';
      case 'small':
      default:
        return '20px';`
        
       but for avatar you want 40px. what should be mapping for avatar sizes? and also for font size what should be the mapping? i assume for height and width medium is 40px and for font size 16px. what about small and big sizes?

Reply:
Good point!
We have discussed that with our design team, you'll find the different sizes and fontSize in the figma: https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=7027%3A100179&mode=dev

So we want 5 sizes: xs, sm, md, lg, xl
Thanks a lot, the help is much appreciated